lib/stableSample.js

```diff
@@ -0,0 +1,77 @@
+const {Cu} = require('chrome');
+Cu.importGlobalProperties(['crypto', 'TextEncoder']);
+
+const {Log} = require('./Log.js');
+
+/**
+ * Map from the range [0, 1] to [0, max(sha256)].
+ * @param  {number} frac A float from 0.0 to 1.0.
+ * @return {string} A string in the sha256 hash space. Will be zero padded to 64
+ *   characters.
+ */
+function fractionToKey(frac) {
+    // SHA 256 hashes are 64-digit hexadecimal numbers. The largest possible SHA
+    // 256 hash is 2^256 - 1.
```

Comment:
I realize now this was copy/pasted from the one in self-repair. Sorry about that. We should fix it in both places.